src/components/general/ParameterWidget.tsx

```diff
@@ -241,16 +237,15 @@ const ParameterWidget = (props: ParameterWidgetProps) => {
   const { parameter } = props;
   const activeScenario = useReactiveVar(activeScenarioVar);
 
-  const [SetParameter, { loading: mutationLoading, error: mutationError }] =
-    useMutation<SetParameterMutation, SetParameterMutationVariables>(
-      SET_PARAMETER,
-      {
-        refetchQueries: 'active',
-        onCompleted: () => {
-          activeScenarioVar({ ...activeScenario });
-        },
-      }
-    );
+  const [SetParameter, { loading: mutationLoading, error: mutationError }] = useMutation<
```

Comment:
I know this is from the old code, but let's change this to `setParameter` to follow our coding style